tests/bluetooth/bsim/mesh/src/test_blob.c

```diff
@@ -27,6 +27,22 @@ static enum {
 	XFER_GET_FAIL = 1
 } msg_fail_type;
 
+static void test_cli_init(void)
+{
+	/* Ensure those test devices will not drift more than
+	 * 100ms for each other in emulated time
+	 */
+	tm_set_phy_max_resync_offset(100000);
+}
+
+static void test_srv_init(void)
+{
+	/* Ensure those test devices will not drift more than
+	 * 100ms for each other in emulated time
+	 */
+	tm_set_phy_max_resync_offset(100000);
```

Comment:
Since it becomes de facto default value for all mesh tests, perhaps we should add it to `bt_mesh_test_cfg` to solve this once and forever (and remove it in all other places)? If I don't miss anything, all our tests call `bt_mesh_test_cfg` initially.